vcluster/configure/vcluster-yaml/sync/from-host/custom-resources.mdx

```diff
@@ -12,31 +12,67 @@ import NamespacedCustomResourcesExample from '../../../../_fragments/sync-from-h
+
+The synced resources are read-only in the virtual cluster. If you modify them in the host cluster, vCluster automatically updates the virtual cluster copies. However, changes made to these resources within the virtual cluster are not persisted.
+
+If you need to sync resources from the virtual cluster to the host cluster instead, see [syncing custom resources to the host cluster](../to-host/advanced/custom-resources).
 
 :::info No need to configure RBAC
 vCluster automatically adds the required cluster RBAC permissions for retrieving the CustomResourceDefinition and syncing the resources from the host to the virtual cluster.
 :::
 
-## Cluster scoped example
+## Cluster-scoped example
 
-To configure vCluster to sync ClusterIssuers from the host cluster (from [cert-manager](https://cert-manager.io/)):
-```yaml title="configure Cluster scoped CRD sync from host"
+Cluster-scoped custom resources exist at the cluster level and are not tied to any specific namespace. These are the simplest to sync because they don't require namespace mapping.
+
+To sync `ClusterIssuers` from the host cluster using [cert-manager](https://cert-manager.io/), configure the resource in your `vcluster.yaml` file:
+
+```yaml title="configure cluster-scoped CRD sync from host"
 sync:
   fromHost:
     customResources:
       clusterissuers.cert-manager.io:
         enabled: true
         scope: Cluster
 ```
-<!-- vale Google.Headings = NO -->
-## Namespace scoped CustomResourceDefinitions
-<!-- vale Google.Headings = YES -->
-By default, this is turned off.
 
-Enabling this allow you to sync namespaced CustomResources from the specified namespaces in the host to the specified namespaces in the vCluster.
-```yaml title="configure Namespace scoped CRD sync from host"
+By default, vCluster does not require you to specify an API version when syncing custom resources. When a CRD has multiple versions and you don't specify a version, vCluster uses the storage version.
+
+If you want to sync a specific version of a CRD that is not the storage version, you can explicitly specify it in the sync configuration.
+
+## Versioned example
+
+To use a specific version of a CustomResourceDefinition (CRD), include the version before the resource type in the configuration, separated by a `/`.
+
+For example, to use version `v1` of the `clusterissuers.cert-manager.io` CRD, specify:
+
+```yaml title="Configure cluster-scoped CRD sync from host with version"
+sync:
+  fromHost:
+    customResources:
+      clusterissuers.cert-manager.io/v1:
+        enabled: true
+        scope: Cluster
+```
+
+:::info Unique CRD versions
+vCluster supports syncing only one version of a custom resource. If you specify multiple versions, the virtual cluster fails to start.
+
+During an upgrade, if a new version of the CRD is required and not yet synced, vCluster automatically updates the CRD to include the new version.
+:::
```

Comment:
Also, the version must exist on the hostCluster

Reply:
Added